modules/publication/jsx/projectFields.js

```diff
@@ -0,0 +1,378 @@
+class EmailElement extends React.Component {
+  constructor() {
```

Comment:
Why not add this to Form.js?

Reply:
yeah i thought about it, might be worth doing later but didn't want to have to put a dependency to another PR for this.

Reply:
which PR are you waiting on?